setup.py

```diff
@@ -5,9 +5,9 @@
 from setuptools import setup
 
 
-MAJOR = 0
-MINOR = 18
-MICRO = 0
+MAJOR = 1
+MINOR = 0
```

Comment:
For a Major/Minor release we bump to the next minor version. So it should be `1.1.0`